src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -801,18 +801,17 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
                 $guard->setArguments([
                     $guardsConfiguration,
                     new Reference('workflow.security.expression_language'),
-                    new Reference('security.token_storage'),
-                    new Reference('security.authorization_checker'),
-                    new Reference('security.authentication.trust_resolver'),
-                    new Reference('security.role_hierarchy'),
+                    new Reference('security.token_storage', ContainerInterface::NULL_ON_INVALID_REFERENCE),
+                    new Reference('security.authorization_checker', ContainerInterface::NULL_ON_INVALID_REFERENCE),
+                    new Reference('security.authentication.trust_resolver', ContainerInterface::NULL_ON_INVALID_REFERENCE),
```

Comment:
I think these changes will break BC, because we need to change from `"symfony/workflow": "^4.4|^5.0",` to `"symfony/workflow": "^5.0",`